browser/ui/webui/brave_settings_ui.cc

```diff
@@ -141,6 +142,11 @@ void BraveSettingsUI::AddResources(content::WebUIDataSource* html_source,
 
   html_source->AddBoolean("extensionsManifestV2Feature",
                           base::FeatureList::IsEnabled(kExtensionsManifestV2));
+#if defined(TOOLKIT_VIEWS)
```

Comment:
Just double checking we need this check - I think most webui pages aren't included on android (but some are :laughing: )

Reply:
I think we should keep this as long as we include the tabs section from the toolbar.html file. But Android doesn't seem to have brave://settings page, so we might be able to remove the guard in the toolbar.html together? 

@simonhong, I'm wondering your opinion.

Reply:
Same here. When we're confusing something like this, I think we could sync gn's configuration. :)